pkgs/data/themes/plata/default.nix

```diff
@@ -37,7 +42,9 @@ stdenv.mkDerivation rec {
     inkscape
     libxml2
     gnome2.glib.dev
-  ];
+  ]
+  ++ stdenv.lib.optional parallelSupport parallel
```

Comment:
You still have `parallel` in the list above, so I guess building with `parallelSupport = false` may fail.

What's the reason for having a conditional parallel build now?

Reply:
I was blindly adding support for all Makefile flags. This probably doesn't make sense, so I went ahead and reverted back to the previous setup which just always enables parallel builds.